ic-utils/src/interfaces/management_canister/builders.rs

```diff
@@ -48,7 +48,7 @@ pub struct CreateCanisterBuilder<'agent, 'canister: 'agent, T> {
     memory_allocation: Option<Result<MemoryAllocation, AgentError>>,
     freezing_threshold: Option<Result<FreezingThreshold, AgentError>>,
     is_provisional_create: bool,
-    amount: Option<u64>,
+    amount: Option<u128>,
```

Comment:
Please include a note in the [changelog](https://github.com/dfinity/agent-rs/blob/main/CHANGELOG.md)

Reply:
Good catch, thank you!